docs/content/triage-guide.md

```diff
@@ -0,0 +1,37 @@
+# Triage Duty Guide
+
+This guide explains the goals of triage duty and outlines the triage process.
+
+## Goals
+
+1. Identify and properly manage critical issues in a timely manner.
+2. Label and prioritize incoming issues.
+3. Help move along conversations in issues until they are scoped for the backlog and ready for someone to pick up, closed or resolved.
+
+## Steps
+
+1. Add all new issues to the [`Spin Triage` project board](https://github.com/orgs/fermyon/projects/7/) under the `Triage Needed` column.
+
+To do this:
+
+- Go to the [project board](https://github.com/orgs/fermyon/projects/7/) and select `+ Add Item` at the bottom of the `Triage Needed` column.
+- Select `+`
+- Select `Add items to project`
+- In the search bar, filter issues by `is:issue is:open`
+- Select all and push the button `add selected items`
+
+2. For each issue in the `Triage Needed` column, add appropriate labels and move the issue to another column if the issue is ready for a different column (`Investigating / Open for Comment`, `Backlog`, `In progress`) or close.
+
+To do this:
+
+- Determine if the issue is a `bug` or an `enhancement` and label as such. Please bubble up and help resolve any critical bugs as you come across them.
```

Comment:
Sometimes it might be neither a bug or an enhancement, e.g. a question.  Or would we consider a question that isn't already addressed in the documentation to be a bug or enhancement for the documentation?

Reply:
Good point. I think it makes sense to word this:
```
Determine if the issue is a `bug`, `enhancement`, or `question` and label as such. Please bubble up and help resolve any critical bugs as you come across them. If the question exposes a need for an improvement in the docs, please open up an issue in the [developer docs repo](https://github.com/fermyon/developer/issues).
```

Wdyt?

Reply:
added 👍